generators/form/templates/formSimple.js.ejs

```diff
@@ -0,0 +1,50 @@
+<%- include('partials/formHeader', {formNumber: formNumber}); %>
+
+const formFields = {
+  firstName: 'firstName'
+};
+
+const formConfig = {
+  urlPrefix: '/',
+  submitUrl: '/v0/api',
+  trackingPrefix: '<%= trackingPrefix %>',
+  introduction: IntroductionPage,
+  confirmation: ConfirmationPage,
+  formId: '<%= formNumber %>',
+  version: 0,
+  prefillEnabled: true,
+  savedFormMessages: {
+    notFound: 'Please start over to apply for <%= benefitDescription %>.',
+    noAuth: 'Please sign in again to continue your application for <%= benefitDescription %>.'
+  },
+  title: '<%= appName %>',
+  defaultDefinitions: {
+  },
+  chapters: {
+    chapter1: {
```

Comment:
I also think that establishing some kind of naming pattern for keys in the `chapters` object and keys in each chapter's `pages` object would be good. Simply starting the key with `chapter` and `page` works for me. Looking at a config object as simple as [the one for the Opt Out](https://github.com/department-of-veterans-affairs/vets-website/blob/generator-doc-update/src/applications/edu-benefits/0993/config/form.js#L24) I would find it easier to digest if the chapter and page keys were easier to pick out. This is especially true in Opt Out config since the keys are the same for both the chapter and page.

Reply:
On second thought, since the page IDs need to be unique across all the chapters, it would be safer to create a `pages` object at the top of the file, similar to the `formFields`